src/index.ts

```diff
@@ -1,7 +1,13 @@
 import { getInitialBoard } from './Board'
-import { renderBoard, render } from './RenderBoard'
+import { boardToGraph } from './Graph'
+//import { renderBoard, render } from './RenderBoard'
```

Comment:
never commit commented out code. Git lets you find code tat you've removed

Reply:
Comments have been cleaned in the [last commit](https://github.com/marmelab/kuba-integration/commit/52c48e332abaf5397cf6debd1512e2111062ba31)